src/components/functional/ledger-connect-components/ledger-connect-panel.tsx

```diff
@@ -145,7 +144,7 @@ export class LedgerConnectPanel {
     const bodyContent = this.getBodyContent(accountScreenData, confirmScreenData, connectScreenData);
 
     return (
-      <mvx-side-panel isOpen={this.isOpen} side={SidePanelSideEnum.RIGHT} panelClassName="ledger-connect-panel" onClose={this.handleClose.bind(this)}>
+      <mvx-side-panel isOpen={this.isOpen} panelClassName="ledger-connect-panel" onClose={this.handleClose.bind(this)}>
```

Comment:
panelClassName --> class ?

Reply:
We only make it a `class` property if that belongs to the first level of the markup. And in this case, it does not.